Files/Scripts/Options/The Sealed Palace.psm1

```diff
@@ -304,12 +341,15 @@ function CreateTabEquipment() {
     
     # EQUIPMENT #
 
-    CreateReduxGroup   -Tag  "Hitbox"      -Height 4.5 -Text "Equipment Adjustments"
```

Comment:
Deku Stick change addition is good, but do not change anything else.

Reply:
Same reason with OoT.psm1